scheduler/docs/reason-code

```diff
@@ -12,6 +12,8 @@
 02001: REASON_CONTAINER_LIMITATION_DISK
 02002: REASON_CONTAINER_LIMITATION_MEMORY
 02003: Max runtime exceeded
+02004: Task was a straggler
+02005: Container initialization timed out
```

Comment:
"Container initialization timed out" feels closer to 01xxx (e.g. "Scheduling failed on host") than it does 02xxx.  What do you think?

Reply:
Good call, will do

Reply:
Done